packages/contractkit/src/utils/azure-key-vault-client.ts

```diff
@@ -5,6 +5,13 @@ import { BigNumber } from 'bignumber.js'
 import { bigNumberToBuffer, bufferToBigNumber, makeCanonical, Signature } from './signature-utils'
 import { publicKeyPrefix, recoverKeyIndex, sixtyFour, thirtyTwo } from './signing-utils'
 
+export enum AzureKeyVaultSigningAlgorithm {
+  ECDSA256 = 'ECDSA256',
+  ES256K = 'ES256K',
```

Comment:
Let's change the default to `ES256K` since `ECDSA256` isn't compatible with the HSMs anymore

Reply:
Or does this only affect keys created after a certain date?

Reply:
> Keys created with the preview curve name SECP256K1 can only be signed with the preview algorithm name, which happens to be ECDSA256 (not ES256K). It's just a name change. Internally they do the exact same thing.
I see.. can we instead fallback to the old curve if the new one fails?

Reply:
Yeah that's one possibility. I was thinking that the cleanest way would be to use the API to get the name of the curve for that specific key from Azure and then based on that determine which algo name to use. I'll investigate this a bit today to see if there's a clean solution. I feel like fallback to other name would be some legacy weird behaviour. 
Querying the curve type would also allow us to improve DevEx by throwing more appropriate errors if the curve is not what we expect.